migrations/migrate-to-7-0-0.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
🚀

Reply:
- I will remove the `pipefail` to support more shells
- Mac was not in scope. Do you see a chance to support the script on a Mac as well? Otherwise Mac users have to start a Docker container with Alpine, ... and run the script there.